cwltool/sandboxjs.py

```diff
@@ -23,6 +23,19 @@ class JavascriptException(Exception):
 have_node_slim = False
 
 
+def check_js_threshold_version(working_alias):
+    # type: (str) -> bool
+    """ parse node version: 'v4.2.6\n' -> ['4', '2', '6'],
+    https://github.com/nodejs/node/blob/master/CHANGELOG.md#nodejs-changelog """
+
+    v1, v2, v3 = [int(v) for v in subprocess.check_output(
```

Comment:
@mr-c after lot's of debugging, I've come to understand that calling ``subprocess.check_output`` is the reason for failure. can I not invoke ``subprocess`` here?